test/data/ansible/check_ostree.yaml

```diff
@@ -45,28 +45,30 @@
     - set_fact:
         checking_stage: "{{ result_stage.stdout }}"
 
+    # Remove this check because sometimes onboarding happens too fast and /boot/device-credentials will be gone,
```

Comment:
We have to keep this test because this case is the only case we check FDO onboarding. We have to find a way to make it stable.